### PR TITLE
Week 7

### PR DESCRIPTION
**Pull request by jerbear4**

### CODE REVIEW

**Comment by PythonCHB** [comment]
Do you have specific questions? we will try to help!

But I'll see what I see in your code, too.

File: students/jbearer/session07/ex40.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env/python3
```
Comment (PythonCHB) [inline students/jbearer/session07/ex40.py]: nice straightforward example -- does it make sense?

File: students/jbearer/session07/html_render/html_render.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    def __init__(self, content=None):
+        self.content = []
+        if content:
+            self.content.append(content)
+
+    def append(self, content):
+        self.content.append(content)
+
+    def render(self, out_file, ind=''):
+        out_file.write("<{}>\n".format(self.tag))
```
Comment (PythonCHB) [inline students/jbearer/session07/html_render/html_render.py]: exactly! nice.

File: students/jbearer/session07/html_render/test_html_render.py [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+    e.render(outfile)
+
+    outfile.seek(0)
+    file_contents = outfile.read()
+
+    print(file_contents)
+
+    assert('This is some text') in file_contents
+    assert('This is some more text') in file_contents
```
Comment (PythonCHB) [inline students/jbearer/session07/html_render/test_html_render.py]: you should probably check that the <p> and </p> have been rendered correctly, too.

File: students/jbearer/session07/html_render/html_render.py (hunk 2 above)
Comment (jerbear4) [inline students/jbearer/session07/html_render/html_render.py]: This was your code. I was trying to figure out how it worked.

Comment (PythonCHB) [inline students/jbearer/session07/html_render/html_render.py]: I thought I liked it :-) -- does it make sense?

**Pull request merged**